docs/source/notebooks/wps_rerank_demo.ipynb

```diff
@@ -13,15 +13,14 @@
   },
```

Comment:
<p>The local <code>url</code> definition here is breaking the tests.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pacificclimate/chickadee/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>